src/index.ts

```diff
@@ -78,11 +78,11 @@ function execCommand(
 		const process = child_process.spawn(command, args, { cwd });
 
 		process.stdout.on("data", (data) => {
-			core.info(data);
+			core.info(String(data));
```

Comment:
While this is not currently presenting a breaking issue, its also treated as a string, and is susceptible to the same problem in the future if not changed.